common/dbconnector.cpp

```diff
@@ -510,6 +529,7 @@ void DBConnector::select(DBConnector *db)
 DBConnector::DBConnector(const DBConnector &other)
     : RedisContext(other)
     , m_dbId(other.m_dbId)
+    , m_dbName(other.m_dbName)
```

Comment:
```suggestion
    , m_dbName(other.m_dbName, other.m_namespace)
```

Do we want the namespace too?